tests/zquantum/core/circuits/symbolic/translations_test.py

```diff
@@ -14,126 +9,33 @@
 
 
 @pytest.mark.parametrize(
-    "sympy_expression, quil_expression",
+    "sympy_expression",
     [
-        (sympy.Symbol("theta"), quil.Parameter("theta")),
-        (
-            sympy.Mul(sympy.Symbol("theta"), sympy.Symbol("gamma"), evaluate=False),
-            quil.Parameter("theta") * quil.Parameter("gamma"),
-        ),
-        (sympy.cos(sympy.Symbol("theta")), quilatom.quil_cos(quil.Parameter("theta"))),
-        (
-            sympy.cos(2 * sympy.Symbol("theta")),
-            quilatom.quil_cos(2 * quil.Parameter("theta")),
-        ),
-        (
-            sympy.exp(sympy.Symbol("x") - sympy.Symbol("y")),
-            quilatom.quil_exp(quil.Parameter("x") - quil.Parameter("y")),
-        ),
-        (
-            sympy.Add(
-                sympy.cos(sympy.Symbol("phi")),
-                sympy.I * sympy.sin(sympy.Symbol("phi")),
-                evaluate=False,
-            ),
-            quilatom.quil_cos(quil.Parameter("phi"))
-            + 1j * quilatom.quil_sin(quil.Parameter("phi")),
-        ),
-        (
-            sympy.Add(
-                sympy.Symbol("x"),
-                sympy.Mul(sympy.Symbol("y"), (2 + 3j), evaluate=False),
-                evaluate=False,
-            ),
-            quil.Parameter("x") + quil.Parameter("y") * (2 + 3j),
-        ),
-        (
-            sympy.cos(sympy.sin(sympy.Symbol("tau"))),
-            quilatom.quil_cos(quilatom.quil_sin(quil.Parameter("tau"))),
-        ),
-        (
-            sympy.Symbol("x") / sympy.Symbol("y"),
-            quil.Parameter("x") / quil.Parameter("y"),
-        ),
-        (
-            sympy.tan(sympy.Symbol("theta")),
-            quilatom.quil_sin(quil.Parameter("theta"))
-            / quilatom.quil_cos(quil.Parameter("theta")),
-        ),
-        (2 ** sympy.Symbol("x"), 2 ** quil.Parameter("x")),
-        (
-            sympy.Symbol("y") ** sympy.Symbol("x"),
-            quil.Parameter("y") ** quil.Parameter("x"),
-        ),
-        (sympy.Symbol("x") ** 2, quil.Parameter("x") ** 2),
-        (
-            sympy.sqrt(sympy.Symbol("x") - sympy.Symbol("y")),
-            quilatom.quil_sqrt(quil.Parameter("x") - quil.Parameter("y")),
-        ),
-        (
-            -5 * sympy.Symbol("x") * sympy.Symbol("y"),
-            -5 * quil.Parameter("x") * quil.Parameter("y"),
-        ),
+        sympy.Symbol("theta"),
+        sympy.Symbol("theta") * sympy.Symbol("gamma"),
+        sympy.cos(sympy.Symbol("theta")),
+        sympy.cos(2 * sympy.Symbol("theta")),
+        sympy.exp(sympy.Symbol("x") - sympy.Symbol("y")),
+        sympy.Add(
```

Comment:
Previously we used `sympy.Add` with `evaluate=False` to preserve the ordering of the arguments. This was needed so that comparison to pyquil expression would succeed. Maybe this is no longer needed, and we could just use + operator?